include/rocksdb/status.h

```diff
@@ -457,10 +457,10 @@ class Status {
   Code code_;
   SubCode subcode_;
   Severity sev_;
-  const char* state_;
 #ifdef ROCKSDB_ASSERT_STATUS_CHECKED
   mutable bool checked_ = false;
 #endif  // ROCKSDB_ASSERT_STATUS_CHECKED
+  const char* state_;
```

Comment:
Why we move the line here?

Reply:
move the line here eliminate paddings and reduces sizeof(Status) by 8 bytes when ROCKSDB_ASSERT_STATUS_CHECKED is defined.